helm/ffc-template-node/values.yaml

```diff
@@ -0,0 +1,43 @@
+environment: development
+name: ffc-template-node
+namespace: ffc-demo
+workstream: ffc-demo
+image: ffc-template-node
+
+labels: {}
+
+containerConfigMap:
+  name: ffc-template-node-config-map
+
+deployment:
+  restartPolicy: Always
+  priorityClassName: low
```

Comment:
Suggest this is set as `default`, not `low`.  Most services will use the default priority profile.  It's just services like the demo service we've kept as `low` as we're happy for it to be killed before other services when resources are stretched.

Reply:
Agreed, done